doc/source/cluster/kubernetes/examples/rayjob-batch-inference-example.md

```diff
@@ -34,20 +33,19 @@ This example uses four [Nvidia T4](https://cloud.google.com/compute/docs/gpus#nv
 ## Step 1: Install the KubeRay Operator
 
 Follow [this document](kuberay-operator-deploy) to install the latest stable KubeRay operator from the Helm repository.
-
-It should be scheduled on the CPU pod.
+The KubeRay operator Pod must be on the CPU node if you have set up the taint for the GPU node pool correctly.
 
 ## Step 2: Submit the RayJob
 
-Create the RayJob custom resource. The RayJob spec is defined in [ray-job.batch-inference.yaml](https://github.com/ray-project/kuberay/blob/v1.0.0/ray-operator/config/samples/ray-job.batch-inference.yaml).
+Create the RayJob custom resource with [ray-job.batch-inference.yaml](https://github.com/ray-project/kuberay/blob/v1.0.0/ray-operator/config/samples/ray-job.batch-inference.yaml).
 
 Download the file with `curl`:
 
 ```bash
 curl -LO https://raw.githubusercontent.com/ray-project/kuberay/v1.0.0/ray-operator/config/samples/ray-job.batch-inference.yaml
 ```
 
-Note that the `RayJob` spec contains a spec for the `RayCluster` that is to be created for the job. For this tutorial, we use a single-node cluster with 4 GPUs.  For production use cases, we recommend using a multi-node cluster where the head node does not have GPUs, so that Ray can automatically schedule GPU workloads on worker nodes and they won't interfere with critical Ray processes on the head node.
+Note that the `RayJob` spec contains a spec for the `RayCluster` that is to be created for the job. For this tutorial, we use a single-node cluster with 4 GPUs. For production use cases, we recommend using a multi-node cluster where the head node doesn't have GPUs, so that Ray can automatically schedule GPU workloads on worker nodes and they won't interfere with critical Ray processes on the head node.
```

Comment:
```suggestion
Note that the `RayJob` spec contains a spec for the `RayCluster` that is to be created for the job. This tutorial uses a single-node cluster with 4 GPUs. For production use cases, use a multi-node cluster where the head node doesn't have GPUs, so that Ray can automatically schedule GPU workloads on worker nodes which won't interfere with critical Ray processes on the head node.
```

Reply:
Can you change "is to be created for the job" to active voice? I don't know who is doing the creating so I don't have a quick suggestion for you.

Reply:
I directly remove "that is to be created for the job.". It is not important.